pkg/vulnsrc/osv/osv.go

```diff
@@ -169,6 +169,9 @@ func (vs VulnSrc) commit(tx *bolt.Tx, eco ecosystem, entry Entry) error {
 
 					// e.g. {"introduced": "1.2.0}, {"fixed": "1.2.5}
 					vulnerable = fmt.Sprintf("%s, <%s", vulnerable, event.Fixed)
+				case event.LastAffected != "":
```

Comment:
We should add a comment about the following spec defined [here](https://ossf.github.io/osv-schema/#requirements).

>Entries in the events array can contain either last_affected or fixed events, but not both